### DISCUSSION TITLE
Benchmarking domain-specific research workflow solutions at an international level

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#37] [Posters for #RSEAA2023]
This is a [presentation to explain how multiple organisations could work with domain researchers](https://figshare.com/articles/presentation/Benchmarking_domain-specific_research_workflow_solutions_at_an_international_level/21259425) to make it easier to understand, discuss, and communicate between research support staff and researchers.

[A wiki of domain level tutorials](https://github.com/Research-Data-Workflows/getting-started-guide/wiki) has been started that demonstrates what it could look like.

And this is a [student presentation using data that was used to benchmark across older HPCs from 2016](https://figshare.com/articles/presentation/Data_Science_Industry_Project_S1_Presentation/23807151).

**Comment by richarddushime** [comment]
I really Liked  This presentation , it offers a comprehensive deep dive into the significance of benchmarking in domain-specific research workflows. The emphasis on reproducibility and performance assessment provides a clear picture of the intricacies involved in ensuring consistency and efficiency. Additionally, the differentiation between reproducibility benchmarking and performance comparison offers an intriguing perspective. The challenges faced by various stakeholder groups such as HPC System Administrators, Research Librarians, and eResearch Managers are articulated well, underscoring the importance of foundational support.  Great Work @rowlandm

**Comment by Aleem2** [comment]
**FAIR for Jupyter Notebooks: A practical guide**

[Learn why and how you should make your Jupyter Notebooks FAIR
](https://doi.org/10.6084/m9.figshare.24135960.v1)
Jupyter Notebooks have been widely used in research and data science [1]. But despite this, there are several challenges in making them findable, accessible, interoperable and reusable (FAIR) to the benefit of both the creators and users. A plausible approach is to apply the FAIR Principles to ensure the appropriate use of Jupyter Notebooks. However, the FAIR Principles are high-level aspirations and can be difficult to apply in practice. Also, translating the FAIR principles into practice will vary for each discipline.

This resource is also available as [ARDC web resource](https://ardc.edu.au/resource/fair-for-jupyter-notebooks-a-practical-guide/) and [github repository](https://github.com/Aleem2/FAIR4JupyterNotebook) as an example, and[ workflow](https://github.com/Aleem2/FAIR4JupyterNotebook/blob/main/Workflow-FAIR4JN.md) for quick adoption.

**Reply by rowlandm** [reply]
Hi Aleem - could you put this as a brand new poster please?

[Go to this page](https://github.com/RSEAA/RSEAA.github.io/discussions/categories/posters-for-rseaa2023) and then click on New Discussion. Let me know if you can't...

**Reply by Aleem2** [reply]
Thanks Rowland, Posted it.
